example.wlk

```diff
@@ -1,6 +1,98 @@
-/** First Wollok example */
-object wollok {
-	method howAreYou() {
-		return 'I am Wolloktastic!'
-	}
+
+object paquete {
+  var estaPago = true
+  //getter 
+  method estaPago(){
+    return estaPago
+  }
+  //setter 
+  method estadoDePaquete(_estaPago){
+    estaPago = _estaPago
+  }
 }
+
+object puenteDeBrookyn {
+  method dejaPasar(mensajero){
+    return mensajero.pesoTotal() < 1000 && paquete.estaPago()
+  }
+}
+
+object laMatrix{
+  method dejaPasar(mensajero){
+    return mensajero.puedeComunicarse() && paquete.estaPago()
+  }
+}
```

Comment:
Estaría bueno evitar el uso de la variable global "paquete" en este método. Acá está medio simple porque trabajamos con un único paquete. Pero en el momento en que haya un segundo paquete (uno que no esté pago por ejemplo), te trae problemas. Para no usar la referencia global tendrías que pensar si el paquete es algo que queres que recuerde el destino (tener atributo "paqueteAEntregar" por ejemplo), o si es algo para recibir por parametro: `method dejarPasar(mensajero, paqueteAEntregar)`  que podés interpretarlo como si deja pasar al mensajero que va a entregar ese paquete.

Igual la mejor solución es arrancar con el paquete para peguntarle `paquete.puedeSerEntregado(destino, mensajero)` y ahí adentro devolver `return self.estaPago() and destino.dejaPasar(mensajero)`